crates/modelardb_storage/src/optimizer/model_simple_aggregates.rs

```diff
@@ -292,6 +282,34 @@ fn try_new_aggregate_exec(
     )?))
 }
 
+/// Return [`()`] if no predicates have been pushed to `grid_exec_child`, otherwise
```

Comment:
Maybe change to "Return [`Ok`]" since it seems we normally write that when returning an empty result.

Reply:
Fixed.